okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -94,9 +94,9 @@
 
     static {
         dispatcher = new Dispatcher(executionExecutor);
-        dispatcher.setMaxRequests(256);
+        dispatcher.setMaxRequests(Integer.MAX_VALUE);
```

Comment:
Probably want a comment on why it's Integer.MAX_VALUE (mention the concurrency limiters) before we merge.

Reply:
Alright, makes sense. I'm taking over from @gmaretic as he's on leave - will push this today

Reply:
@CRogers I've added the comment on #1174, as I don't have write permissions to the repository. If we're happy with that we should merge that into this and then this to develop. Thanks!